setup.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import absolute_import, division, print_function
 
+import setuptools
```

Comment:
setuptools monkey patches distutils.  Might want to check with @asmeurer to ensure that this is ok.